docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -51,3 +51,6 @@ connector:
       JDBC query. Using a large timeout can potentially result in more detailed
       dynamic filters. However, it can also increase latency for some queries.
     - ``20s``
+  * - ``postgresql.include-system-tables``
+    - Supports access to postgres system catalog tables(pg_catalog schemas).
+    - ``false``
```

Comment:
Please update the commit title as "Document postgresql.include-system-tables config property"
https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages